chapter-13-browser-based-apps/backend-for-frontend/oauth-agent/src/lib/cookieEncrypter.ts

```diff
@@ -77,7 +77,7 @@ function decryptCookie(encKeyHex: string, encryptedbase64value: string): string
     }
 }
 
-function getEncryptedCookie(options: CookieSerializeOptions, value: string, name: string, encKey: string): string {
+function getEncryptedCookie(options: SerializeOptions, value: string, name: string, encKey: string): string {
```

Comment:
There was a critical security warning so I used a 1.x version where the object is renamed